examples/research_projects/diffusion_dpo/train_diffusion_dpo.py

```diff
@@ -858,12 +864,19 @@ def collate_fn(examples):
                 accelerator.unwrap_model(unet).enable_adapters()
 
                 # Final loss.
-                scale_term = -0.5 * args.beta_dpo
-                inside_term = scale_term * (model_diff - ref_diff)
-                loss = -1 * F.logsigmoid(inside_term).mean()
```

Comment:
Can we guard this loss as well? I don't see the `scale_term` being used here. Maybe elaborate a bit?

Reply:
right, so scale term no is the `beta_dpo * .5 = 2500` and the neg goes in the bracket so `(ref_diff - model_diff)` meaning the loss becomes `-1 * F.logsigmoid(args.beta_dpo * (ref_diff - model_diff))` so the orignal loss stays the same